controller_list:
  - name: ""
    action_ns: joint_trajectory_action
    type: FollowJointTrajectory
    joints: [joint_1, joint_2, joint_3, joint_4, joint_5, joint_6]

# controller_list:
#   - name: "robot_1"
#     action_ns: joint_trajectory_action
#     type: FollowJointTrajectory
#     joints: [robot_1_joint_1, robot_1_joint_2, robot_1_joint_3, robot_1_joint_4, robot_1_joint_5, robot_1_joint_6]
#   - name: "robot_2"
#     action_ns: joint_trajectory_action
#     type: FollowJointTrajectory
#     joints: [robot_2_joint_1, robot_2_joint_2, robot_2_joint_3, robot_2_joint_4, robot_2_joint_5, robot_2_joint_6]
